src/main/webapp/dev/js/common/updateCharLeft.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Updates the number of characters left in the text area
+ * @param textArea - Text area for which the number of characters are to be counted
+ * @param letterCountId - Id of Label to display length of text area
```

Comment:
Naming not very accurate, maybe something as follows:
` * @param remainingCharactersElementId - The ID of the HTML element that displays the value of the remaining characters.`